### ISSUE TITLE
Keyboard Focus option doesn't work

### ISSUE THREAD

**Issue by esmoak** [F4uzan/HgLauncher#36]
**What's the issue?**
Selecting Keyboard Focus doesn't focus the keyboard when the app panel is pulled down. 

**To reproduce…**
1. Select Keyboard Focus option
2. Pull app panel down
3. No keyboard pops up without clicking in the search box
4. Attempted to change default keyboard and restart phone, and this didn't fix the issue either. 

**Expected behaviour**
Keyboard pops up without clicking in the search box. 

**What did you see instead?**
Nothing 

**Device information (please complete the following information):**
Android: 8.0.0
Model: BND-L34
Manufacturer: HUAWEI
ID: HUAWEIBND-L34
Hg version: 1.4.2.1 from F-Droid

I don't think any of your recent commits addressed this. 

Keep being awesome. The gestures really set this apart from other FOSS launchers. Thanks for all you've done.

**Comment by F4uzan** [comment]
I have noticed this too over the past few releases. What seemed to work for me was to pull down the panel again; then the keyboard would respond to the change in focus.

I have a good idea for where the issue is occurring. I might roll out a fix for it in the next release.

Thanks for the report!

**Comment by F4uzan** [comment]
This should be all good with 8c70dbd42ba058bc0d291392f916465504e865af. Closing this for now.
